src/totango.js

```diff
@@ -417,7 +421,7 @@ async function closed({ issue }) {
   console.log('Issue was closed');
   // Check to see if the issue has a task associated with it
   // If it does, and the task is not already closed, close the task
-  let task_id = await issue_has_task_id({issue: issue});
+  let task_id = issue_has_totango_id({body});
```

Comment:
Looks like this is trying to pass a `body` which is undefined within this function